src/app/profiles/profile-detail/profile-detail.component.ts

```diff
@@ -119,9 +119,12 @@ export class ProfileDetailComponent implements OnInit {
       this.profileForm.controls.mebxPassword.disable()
       this.profileForm.controls.mebxPassword.setValue(null)
       this.profileForm.controls.mebxPassword.clearValidators()
+      this.profileForm.controls.generateRandomMEBxPassword.setValue(1)
```

Comment:
why `1` ?

Reply:
Essentially, I'm setting the checkbox for the MEBX password to yes before disabling it. The 1 is for checking the checkbox.